src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax;
+
+import static core.basesyntax.db.Storage.people;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl(new StorageDaoImpl());
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setPassword("passwordTest123");
+        user.setLogin("testLogin");
+        user.setAge(21);
+    }
+
+    @Test
+    void userAdd_OK() {
+        assertEquals(user, registrationService.register(user));
+    }
+
+    @Test
+    void registerUserIsNull_notOK() {
```

Comment:
```suggestion
    void register_userIsNull_notOk() {
```
And everywhere below